### DISCUSSION TITLE
please resend the activation email

### DISCUSSION THREAD

**Discussion by altafsamnani** [primefaces/community#2558] [PrimeVue Blocks]
To the Primevue Team Board Administrator, 

Can you please resend the activation email for development@osec.nl, 
or 
if you can let us know the username for deveopment@osec.nl that would be great

we cant login because username is not matched.

We want to buy paid primeblock version

Looking forward to hear from you!

**Comment by cagataycivici** [comment]
Hi, activated the account.

development@osec.nl

User: developer.osec
